.gitignore

```diff
@@ -18,6 +18,7 @@ vendor
 ehthumbs.db
 Thumbs.db
 .thumbsdb
+composer.lock
```

Comment:
We shouldn't gitignore the composer.lock file, since it ensures that the same version of dependencies are installed everywhere, and it helps with caching.

Reply:
You are right, I removed it while testing one approach recommended by WordPress core.